docs/parameters.md

```diff
@@ -13,18 +56,61 @@ When selecting the bounds for a parameter it is important to remember that all v
     max: 4000
 ```
 
-You would consume this parameter in a `spec.containers[].resources` patch of:
+This parameter would be consumed in a patch of a Deployment's `spec.template.spec.containers[].resources` field:
 
 ```yaml
-  requests:
-    cpu: "{{ .Values.cpu }}m"
+  patches:
+  - patch: |
+      spec:
+        template:
+          spec:
+            containers:
+            - name: container-name
+              resources:
+                requests:
+                  cpu: "{{ .Values.cpu }}m"
 ```
 
-## Parameter Manipulation
+Note the inclusion of an `m` to specify millicpus.
 
-All parameters are suggested as integer values, sometimes it is necessary to manipulate a value to consume it in a patch. Patches are evaluated as [Go templates](https://golang.org/pkg/text/template/) with the added [Sprig](http://masterminds.github.io/sprig/) template functions. Additional template functions are also available:
+### Parameter Manipulation
+
+Sometimes it is necessary to manipulate a value to consume it in a patch. To support this, patches are evaluated as [Go templates](https://golang.org/pkg/text/template/), with [Sprig](http://masterminds.github.io/sprig/) template functions included to support a variety of operations. Additional template functions are also available:
 
 - **percent**
   Return the integer percentage.
 
-  `percent 9 50` will return `"4"`
+  `percent 9 50` will return `"4"`.
+
+## Parameter Constraints
+
+**Constraints** restrict the assignments that parameters may take relative to one another. Constraints are optional and used in addition to the bounds on individual parameters like `min` or `max`.
+
+Red Sky Ops supports two types of constraints: **Order** and **Sum**.
+
+### Order Constraint
+
+The order constraint requires that one integer parameter be strictly larger than another. For example, you can use an order constraint to ensure that a container's maximum replicas value is always greater than it's minimum replicas:
```

Comment:
Do order constraints only apply to integers, or do we do any kind of funky string comparison? Also, is it accurate to say "strictly larger"?